src/devtools/client/webconsole/actions/input.ts

```diff
@@ -143,18 +126,8 @@ export function eagerEvalExpression(expression: string): UIThunkAction {
       return null;
     }
 
-    const selectedFrame = getSelectedFrame(getState());
-    if (!selectedFrame) {
-      return;
-    }
```

Comment:
removing the early returns is sufficient. Everything else is cleanup to remove ThreadFront from as many places as possible